chromium_src/chrome/browser/ui/webui/side_panel/bookmarks/bookmarks_side_panel_ui.cc

```diff
@@ -22,7 +22,7 @@ BookmarksSidePanelUIConfig::BookmarksSidePanelUIConfig()
                                   chrome::kChromeUIBookmarksSidePanelHost) {}
 
 bool BookmarksSidePanelUIConfig::IsPreloadable() {
-  return true;
+  return false;
```

Comment:
I think it might be better to remove the override for this method actually- so that it falls back to the upstream one

https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/webui/side_panel/bookmarks/bookmarks_side_panel_ui.cc;l=69-75;drc=d44bc30f6ee03d76c76210bcdad260a226693f08

Not sure why we added it originally

Reply:
Good call!

Reply:
ah - yes, I missed this 😓 sorry

`BookmarksSidePanelUIConfig` is ours and we're implementing by inheriting from `DefaultTopChromeWebUIConfig<>` the same way as upstream, but not using the upstream code. This would be different behavior.

Maybe a follow up could be exploring if its possible to use upstream `BookmarksSidePanelUIConfig` at all?